experiment/measurer.py

```diff
@@ -355,42 +360,48 @@ def run_cov_new_units(self):
         coverage_binary = get_coverage_binary(self.benchmark)
         crashing_units = run_coverage.do_coverage_run(coverage_binary,
                                                       self.corpus_dir,
-                                                      self.sancov_dir,
+                                                      self.profraw_file,
                                                       self.crashes_dir)
 
         self.UNIT_BLACKLIST[self.benchmark] = (
             self.UNIT_BLACKLIST[self.benchmark].union(set(crashing_units)))
 
-    def merge_new_pcs(self) -> List[str]:
-        """Merge new pcs into |self.covered_pcs_filename| and return the list of
-        all covered pcs."""
-
-        # Create the covered pcs file if it doesn't exist yet.
-        if not os.path.exists(self.covered_pcs_filename):
-            filesystem.write(self.covered_pcs_filename, '')
-
-        with open(self.covered_pcs_filename, 'r+') as file_handle:
-            current_pcs = set(
-                pc.strip() for pc in file_handle.readlines() if pc.strip())
-            sancov_files = glob.glob(os.path.join(self.sancov_dir, '*.sancov'))
-            if not sancov_files:
-                self.logger.error('No sancov files.')
-                return list(current_pcs)
-
-            self.logger.info('Sancov files: %s.', str(sancov_files))
-            new_pcs = set(sancov.GetPCs(sancov_files))
-            all_pcs = sorted(list(current_pcs.union(new_pcs)))
-            # Sort so that file doesn't change if PCs are unchanged.
-            file_handle.seek(0)
-            file_handle.write('\n'.join(all_pcs))
-        return all_pcs
-
-    def get_current_pcs(self) -> Set[str]:
-        """Get the current pcs covered by a fuzzer."""
-        with open(self.covered_pcs_filename) as file_handle:
-            current_pcs = set(
-                pc.strip() for pc in file_handle.readlines() if pc.strip())
-        return current_pcs
+    def get_current_coverage(self) -> int:
+        """Get the current number of lines covered"""
+        with open(self.summary_file) as summary:
+            coverage_info = json.loads(summary.readlines()[-1])
```

Comment:
For bloaty, the first line would be a warning info and the second line is the json we want. Only did this way to skip that warning for bloaty since the warning doesn't affect the coverage data as far as I could see. Or should I treat bloaty as a problematic benchmark and not generating anything from it?